docs/diagrams/bedrock_basket_ctms.md

```diff
@@ -0,0 +1,145 @@
+# Basket Integration
+
+## VPN Waiting List
+
+```mermaid
+sequenceDiagram
+    autonumber
+    actor Visitor
+    Visitor->>+mozilla.org: Visits landing page
+    activate mozilla.org
+    mozilla.org-->>-Visitor: Renders form [token]
+    deactivate mozilla.org
+
+    Visitor->>mozilla.org: Submit form [email, token]
```

Comment:
I _believe_ this is a "subscribe" event; hitting this code in basket:
https://github.com/mozmeao/basket/blob/ba2abdecbddd0a186a716398b33b4dff0a775d73/basket/news/views.py#L468

Reply:
So that would be for the workflow of Relay where the form is posted on basket directly?

Reply:
I believe this is the route they were planning on taking